ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
+
+inputs:
+  - name: dp-dataset-api
```

Comment:
I believe this be `dp-api-clients-go`

Reply:
Also wondering whether it is worth upgrading library to use latest go version - i think it is now on 1.16.5